lib/node_modules/@stdlib/math/base/special/logit/src/main.c

```diff
@@ -0,0 +1,50 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2022 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#include "stdlib/math/base/special/logit.h"
+#include "stdlib/math/base/assert/is_probability.h"
+#include "stdlib/math/base/assert/is_nan.h"
+#include "stdlib/math/base/special/ln.h"
+#include "stdlib/constants/float64/pinf.h"
+#include "stdlib/constants/float64/ninf.h"
+
+/**
+* Evaluates the logit function.
+*
+* @param x    input value
+* @return	  output value
+*
+* @example
+* double out = stdlib_base_logit( 0.5 );
+* // returns ~0.405
```

Comment:
In the REPL,

```
In [1]: base.logit(0.5)
Out[1]: 0
```

So this return value does not look correct to me.

Reply:
This is incorrect in `main.c`.